### PR TITLE
refac: Extract OutgoingMessages.IntegrationTests project

### PR DESCRIPTION
**Pull request by MWO1024**
## Description

billede

billede

Note that `AggregatedTimeSeriesRequestAcceptedToAggregationResultTests` has been deleted and merged with `CalculatedQuantityQualityMapperTests`. Additionally, the tests of `QuantityQualitiesFactor.CreateQuantityQuality` have been extracted to its own class.

billede

## References

https://app.zenhub.com/workspaces/mosaic-60a6105157304f00119be86e/issues/gh/energinet-datahub/team-mosaic/311

## Checklist
- [x] Should the change be behind a feature flag?
- [x] Can the feature be meaningfully disabled or circumvented if there are issues (e.g., database-breaking changes)?
  - `git revert`
- [x] Has it been considered whether data is being delivered to the wrong actor?
- [ ] Subsystem test executed (dev_002/dev_003)
- [x] Is there time to monitor state of the release to Production?
- [x] Reference to the task

### CODE REVIEW

File: source/OutgoingMessages.IntegrationTests/Fixtures/IntegrationTestFixture.cs [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+using Energinet.DataHub.BuildingBlocks.Tests.Database;
+using Energinet.DataHub.Core.FunctionApp.TestCommon.Azurite;
+using Energinet.DataHub.Core.FunctionApp.TestCommon.Configuration;
+using Energinet.DataHub.Core.FunctionApp.TestCommon.Databricks;
+using Energinet.DataHub.EDI.BuildingBlocks.Domain.Models;
+using HttpClientFactory = Energinet.DataHub.Core.FunctionApp.TestCommon.Databricks.HttpClientFactory;
+
+namespace Energinet.DataHub.EDI.OutgoingMessages.IntegrationTests.Fixtures;
+
+public class IntegrationTestFixture : IDisposable, IAsyncLifetime
```
Comment (RasmusGraabaek) [inline source/OutgoingMessages.IntegrationTests/Fixtures/IntegrationTestFixture.cs:25]: OutgoingMessagesFixture

File: source/OutgoingMessages.IntegrationTests/Fixtures/IntegrationTestFixture.cs [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    public IntegrationTestFixture()
+    {
+        IntegrationTestConfiguration = new IntegrationTestConfiguration();
+
+        DatabaseManager = new EdiDatabaseManager("IntegrationTests");
+
+        DatabricksSchemaManager = new DatabricksSchemaManager(
+            new HttpClientFactory(),
+            databricksSettings: IntegrationTestConfiguration.DatabricksSettings,
+            schemaPrefix: "edi_integration_tests");
```
Comment (RasmusGraabaek) [inline source/OutgoingMessages.IntegrationTests/Fixtures/IntegrationTestFixture.cs:38]: Should we include outgoingMessages in the prefix?

**Review by RasmusGraabaek — APPROVED** [review]

File: source/OutgoingMessages.IntegrationTests/OutgoingMessages/SampleData.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Energinet.DataHub.EDI.OutgoingMessages.IntegrationTests.OutgoingMessages;
+
+internal static class SampleData
+{
+    internal static string MeteringPointNumber => "571234567891234568";
+
+    internal static string NewEnergySupplierNumber => "5790000555551";
```
[the remark this answers is not in the document]
Comment (MWO1024) [reply source/OutgoingMessages.IntegrationTests/OutgoingMessages/SampleData.cs:21]: I was moved from the integration project, so in that sense, it isn't new.

File: source/OutgoingMessages.IntegrationTests/Assertions/AssertXmlMessage.cs [hunk 4]
```diff
@@ -12,15 +12,10 @@
...
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
 using System.Xml.Linq;
 using Energinet.DataHub.EDI.BuildingBlocks.Domain.Models;
-using Xunit;
 
-namespace Energinet.DataHub.EDI.IntegrationTests.Assertions;
+namespace Energinet.DataHub.EDI.OutgoingMessages.IntegrationTests.Assertions;
```
[the remark this answers is not in the document]
Comment (MWO1024) [reply source/OutgoingMessages.IntegrationTests/Assertions/AssertXmlMessage.cs:18]: I have added a TODO on my list

**Review by RasmusGraabaek — APPROVED** [review]

**Review by RasmusGraabaek — APPROVED** [review]

**Review by lanni-energinet — APPROVED** [review]

**Pull request merged**